### ISSUE TITLE
edgecast_dns_zone not working

### ISSUE THREAD

**Issue by cosimomeli** [Edgio/terraform-provider-edgecast#162]
Hi,
I'm trying to create a dns zone using the provider, but i get `AddZone: SubmitRequest: sendRequest failed (HTTP StatusCode:500): {"Message":"Operation Error. Contact Administrator"}`
I can create the same zone with the same data just using curl to send the POST for the Add Primary Zone API, so my credentials and the data I'm using are ok.

Resource definition:
```
resource "edgecast_dns_zone" "main" {
  account_number    = "redacted"
  domain_name       = var.zone_name
  status            = 1    # 1: active, 2: inactive
  zone_type         = 1    # 1: Primary zone. This value should always be 1.
  is_customer_owned = true # This value should always be true

  record_txt {
    name = "test"
    ttl = 3600
    rdata = "testtxt"
  }
}
  ```

Tried with Terraform 1.3.9 and 0.13.7 and with provider 1.2.1 and 1.1.1

**Comment by stevenpaz** [comment]
@cosimomeli - apologies for the delay in my response. We have identified a fix that will go in out in v1.2.4. Thank you for reporting this.

**Comment by ssaluja4** [comment]
@cosimomeli  - this issue was addressed with our latest release. 
https://registry.terraform.io/providers/Edgio/edgecast/1.3.0

Please let us know if any issues. Thanks.
